README.md

```diff
@@ -1,22 +1,114 @@
+Shows the total revenue generated from all items on the invoice.
+Allows merchants to update the status of invoice items.
+
+## Admin Dashboard
+
+Shows a header indicating that the user is on the admin dashboard.
+Provides links to the admin merchants index and admin invoices index.
+Lists the top 5 customers based on the number of successful transactions.
+Displays invoices with items that have not yet been shipped, sorted by creation date.
+
+## Admin Merchants
+
+Lists all merchants in the system.
+Allows admins to view merchant details on the merchant show page.
+Enables admins to update merchant information.
+Allows admins to enable or disable merchants.
+Groups merchants by status (enabled or disabled).
+Allows admins to create new merchants with a default status of disabled.
+Lists the top 5 merchants based on total revenue, with links to the merchant show page and total revenue for each merchant.
+Displays the top selling date for each of the 5 top-earning merchants.
+
+## Admin Invoices
+
+Lists all invoices in the system, linking each ID to the admin invoice show page.
+Shows invoice details, including ID, status, created date, and customer's first and last name.
+Lists items on the invoice with item name, quantity, price, and item status.
+Displays the total revenue generated from the invoice.
+Allows admins to update the status of invoices.
+
+## API Consumption
+Uses the Unsplash API to display a logo image at the top of every page.
+Fetches images related to item names on the Merchant Item Show page.
+Shows random photos near the name of the Merchant on the Merchant's Dashboard, with the image refreshing on each page refresh.
+Displays the number of likes for the app logo image.
+Potential Contributions and Refactoring
+
+
+### While the current implementation covers the specified features, there is always room for improvement and contributions from the community. Here are some potential areas for future contributions and refactoring:
+
+Improve UI/UX: Enhance the user interface and user experience for both merchants and admins.
+Additional Features: Add more features such as advanced search, item categories, or shipping tracking for merchants.
+Testing: Increase test coverage with more comprehensive unit and integration tests.
+Code Refactoring: Improve code organization, readability, and maintainability for better long-term scalability.
+Error Handling: Implement robust error handling and feedback mechanisms for a better user experience.
+Performance Optimization: Optimize database queries and API requests to improve application performance.
+Accessibility: Ensure the application meets accessibility standards for all users.
+
+## Links for this project
+
+Project's render site: 
+
+Diagram site: 
+
+Group DTR: https://docs.google.com/document/d/1rITzHzPoXYcNLyEDX3FQwvB98AbPXOpPaZ0PCDH7Foo/edit
+
+## This project's team members
+
+### Artemy Gibson
+
+Github profile: https://github.com/algibson1
```

Comment:
We can format these so that the words `Github profile` contain the link. We should also throw in our LinkedIn profiles :)